app/frontend/actions.ts

```diff
@@ -58,6 +57,7 @@ export default (store: Store) => {
     resetPoolRegTransactionSummary,
     deregisterStakingKey,
   } = poolOwnerActions(store)
+  const {exportJsonWallet} = exportWalletActions(store)
```

Comment:
I think that we can move it into `wallet` file, it seems odd to me to have just single action in `exportWallet` when we already have `wallet` file.

Reply:
Fixed